### PR TITLE
Create Portfolio Projects section and About Me section

### PR DESCRIPTION
**Pull request by Mindsetter**
1. Add HTML and CSS linter to check for errors
2. Add my Recent Works Section to contain my project works
3. Add About Me Section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -67,6 +67,193 @@ <h3 class="my-role">I’m a Software Developer</h3>
...
+            no accounts or sign-ups required. has been the 
+            industry's standard
+          </p>
+            <ul>
+              <li>html</li>
+              <li>boostrap</li>
+              <li>Ruby</li>
+            </ul>
+        </div>
+        <button class="see-project-button" >See Project</button>
```
Comment (muneebulrehman) [inline index.html:126]: The `button` should always have a type attribute. [Why?](https://www.w3schools.com/tags/tag_button.asp)

File: index.html [hunk 2]
```diff
@@ -67,6 +67,193 @@ <h3 class="my-role">I’m a Software Developer</h3>
...
+    <section class="about-me">
+      <div>
+        <h1>About Me </h1>
+        <p>Hello I’m a software developer! I can help you build 
+          a product , feature or website Look through some of 
+          my work and experience! If you like what you see and 
+          have a project you need coded, don’t hestiate to 
+          contact me.
+        </p>
+        <button>Get My Resume</button>
```
Comment (muneebulrehman) [inline index.html:219]: Please add a type attribute on the buttons

File: index.html [hunk 3]
```diff
@@ -67,6 +67,193 @@ <h3 class="my-role">I’m a Software Developer</h3>
...
+          <h3>Skills</h3>
+        </div>
+        <ul>
+          <li>Codekit</li>
+          <li>Github</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
+      </div>
```
Comment (muneebulrehman) [inline index.html:255]: Let's wrap technology categories sections in a `<ul>` tag and make each category a list item. I am sure that would make more sense.

File: index.html [hunk 4]
```diff
@@ -67,6 +67,193 @@ <h3 class="my-role">I’m a Software Developer</h3>
...
+          <img src="image/circle.png" alt="">
+          <h3>Skills</h3>
+        </div>
+        <ul>
+          <li>Codekit</li>
+          <li>Github</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
```
Comment (muneebulrehman) [inline index.html:254]: There is a bug in this section. Here is a snapshot. Please fix it.
issue

File: index.html [hunk 5]
```diff
@@ -67,6 +67,193 @@ <h3 class="my-role">I’m a Software Developer</h3>
...
+      <div>
+        <h1>About Me </h1>
+        <p>Hello I’m a software developer! I can help you build 
+          a product , feature or website Look through some of 
+          my work and experience! If you like what you see and 
+          have a project you need coded, don’t hestiate to 
+          contact me.
+        </p>
+        <button>Get My Resume</button>
+        <hr>
```
Comment (muneebulrehman) [inline index.html:220]: Let's center this `<hr>` in the center. That would look better. 
This is how it looks currently.
issue2

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @Mindsetter , 

Good job so far! I see you have put a lot of effort into this section of the project, but there are still some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Good PR description
- Good CSS 
- Good HTML structure.
CatDrivingSeriousGIF (2)


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Making frequent commits, because it would make your debugging life easier.
- Add a setup section in your readme file to tell users how to clone or set up this project.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (Mindsetter) [reply index.html:255]: Can you please explain this further?

File: index.html (hunk 1 above)
Comment (Mindsetter) [reply index.html]: Fixed

File: index.html (hunk 2 above)
Comment (Mindsetter) [reply index.html]: Fixed

File: index.html (hunk 3 above)
Comment (Mindsetter) [reply index.html]: Fixed

File: index.html (hunk 4 above)
Comment (Mindsetter) [reply index.html:255]: Fixed

**Review by Chimwemwe-127001 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

Hi @Mindsetter, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors :heavy_check_mark:
- You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.
- Repo looks professional :briefcase:
- Solid pull request :scroll:
- High fidelity of the Figma template :mag: 
- Cheers and good luck in your next milestones! 🍀

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**